source/agora/network/NetworkManager.d

```diff
@@ -430,6 +430,10 @@ public class NetworkManager
                     address, &onHandshakeComplete, &onFailedRequest);
             }
         }
+
+        connnectionTask(); // avoid delay
+        this.taskman.setTimer(this.node_config.retry_delay.msecs,
```

Comment:
This is not correct. Previously the calling fiber was yielded, but here you are spawning a new one. You're going to end up with multiple fibers running within this class. This part should not be changed.

Reply:
That's what I was curious about. Thank you for your clear answer.

Reply:
The commit has been deleted.